Neos.Neos/Classes/PendingChangesProjection/ChangeProjection.php

```diff
@@ -490,12 +488,4 @@ private function getChange(
         // We always allow root nodes
         return $changeRow ? Change::fromDatabaseRow($changeRow) : null;
     }
-
-    private function isLiveContentStream(ContentStreamId $contentStreamId): bool
```

Comment:
We never need to know this anymore? Weird if so, I would think this is quite useful to have.